.github/workflows/ci.yaml

```
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.23.10, 1.24.4]
+        go-version: [1.24.7, 1.25.1]
```

You could use 


```suggestion
        go-version: [oldstable, stable]
```

This way, there is no need yo update these files.